### DISCUSSION TITLE
PowerDNS integration

### DISCUSSION THREAD

**Discussion by aopdal** [StackStorm/st2#5640] [Packs] [answered]
Hi, I'm trying to learn PowerDNS and StackStorm. Using the powerdns pack while I'm learning I can't find a way to manage more than one server. Having more than one DNS server is a requirement, and I wonder how is the intended way of usage of the powerdns pack with more than one server. Should all zones be native and everything be replicated using the backend in powerdns?

Best Regards
Arne Opdal

**Comment by nzlosh** [comment, marked as answer]
The current state of the pack doesn't provide profiles, but that would be the way the pack would handle multiple servers.  There has been effort to automate the pack content for powerdns.  There is still work on going, so you should probably create an issue in the powerdns repo requesting multiple server support to be added.
